src/explore-education-statistics-frontend/.env

```diff
@@ -1,3 +1,4 @@
 CONTENT_API_BASE_URL=http://localhost:5010/api
 DATA_API_BASE_URL=http://localhost:5000/api
 NOTIFICATION_API_BASE_URL=http://localhost:7073/api
+GA_TRACKING_ID=
```

Comment:
Presumably adding this to the `.env` file with no value makes no difference to the initialisation of GA because the empty string will be falsy in `if (process.env.GA_TRACKING_ID && ...`?

Just checking 🙂

Reply:
Yup that's correct!  Nick also pointed out that we can add actual values to a .env.local file and keep out of Git